modules/ossm-cert-manager-installation.adoc

```diff
@@ -60,6 +52,20 @@ spec:
     secretName: root-ca
 ----
 +
+====
+[NOTE]
+The namespace of the `selfsigned-root-issuer` issuer and `root-ca` certificate is `cert-manager` because `root-ca` is a cluster issuer, so the cert-manager looks for a referenced secret in its own namespace. Its own namespace is `cert-manager` in the case of the {cert-manager-operator}.
```

Comment:
```suggestion
The namespace of the `selfsigned-root-issuer` issuer and `root-ca` certificate is `cert-manager` because `root-ca` is a cluster issuer, so the cert-manager looks for a referenced secret in its own namespace. The namespace is called `cert-manager` in the case of the {cert-manager-operator}.
```

Reply:
Suggested, because the style guide discourages anthropomorphism.

Reply:
@GroceryBoyJr Good call. I just moved the note without looking that closely.